src/app/view/components/site-selector/dot-site-selector.component.ts

```diff
@@ -94,9 +91,14 @@ export class SiteSelectorComponent implements ControlValueAccessor {
      * @memberof SiteSelectorComponent
      */
     paginateSites(filter = '',  page = 1): void {
-        this.siteService.paginateSites(filter, false, page, this.paginationPerPage).subscribe( sites => {
-            this.sitesCurrentPage = sites;
+        this.paginationService.filter = filter;
+        this.paginationService.getPage(page).subscribe( items => {
+            this.sitesCurrentPage = items;
             this.selectCurrentSite();
+
+            if (!this.totalRecords) {
+                this.totalRecords = this.paginationService.totalRecords;
```

Comment:
I think this could change to this.totalRecords = this.totalRecords | this.paginationService.totalRecords;